CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Update Start button with info text ([PR #2476](https://github.com/alphagov/govuk_publishing_components/pull/2476))
```

Comment:
Could you change either the PR title or this line so they're an exact match, please?

Reply:
Course, done.